misc/config_tools/data/adl-rvp/industry.xml

```diff
@@ -0,0 +1,228 @@
+        <target_vm_id>2</target_vm_id>
+        <target_uart_id>1</target_uart_id>
+    </legacy_vuart>
+    <console_vuart id="0">
+        <base>INVALID_PCI_BASE</base>
+    </console_vuart>
+    <communication_vuart id="1">
+        <base>INVALID_PCI_BASE</base>
+        <target_vm_id>1</target_vm_id>
+        <target_uart_id>1</target_uart_id>
+    </communication_vuart>
+    <pci_devs>
+        <pci_dev/>
+    </pci_devs>
+    <board_private>
+        <rootfs>/dev/nvme0n1p3</rootfs>
+        <bootargs>        rw rootwait console=ttyS0 consoleblank=0 no_timer_check quiet loglevel=3 i915.nuclear_pageflip=1
+    </bootargs>
+    </board_private>
+  </vm>
+  <vm id="1">
+    <vm_type>POST_STD_VM</vm_type>
+    <guest_flags>
+        <guest_flag/>
+    </guest_flags>
+    <cpu_affinity>
+        <pcpu_id>0</pcpu_id>
+        <pcpu_id>1</pcpu_id>
+    </cpu_affinity>
+    <clos>
+        <vcpu_clos>0</vcpu_clos>
+        <vcpu_clos>0</vcpu_clos>
+    </clos>
+    <epc_section>
+        <base>0</base>
+        <size>0</size>
+    </epc_section>
+
+    <legacy_vuart id="0">
+        <type>VUART_LEGACY_PIO</type>
+        <base>COM1_BASE</base>
+        <irq>COM1_IRQ</irq>
+    </legacy_vuart>
+    <legacy_vuart id="1">
+        <type>VUART_LEGACY_PIO</type>
+        <base>INVALID_COM_BASE</base>
+        <irq>COM2_IRQ</irq>
+        <target_vm_id>0</target_vm_id>
+        <target_uart_id>1</target_uart_id>
+    </legacy_vuart>
+    <console_vuart id="0">
+        <base>INVALID_PCI_BASE</base>
+    </console_vuart>
+    <communication_vuart id="1">
+        <base>INVALID_PCI_BASE</base>
+        <target_vm_id>1</target_vm_id>
+        <target_uart_id>1</target_uart_id>
+    </communication_vuart>
+  </vm>
+  <vm id="2">
+    <vm_type>POST_RT_VM</vm_type>
+    <guest_flags>
+        <guest_flag>0</guest_flag>
+    </guest_flags>
+    <cpu_affinity>
+        <pcpu_id>2</pcpu_id>
+        <pcpu_id>3</pcpu_id>
+    </cpu_affinity>
+    <clos>
+        <vcpu_clos>0</vcpu_clos>
+        <vcpu_clos>0</vcpu_clos>
+    </clos>
+    <epc_section>
+        <base>0</base>
+        <size>0</size>
+    </epc_section>
+    <legacy_vuart id="0">
+        <type>VUART_LEGACY_PIO</type>
+        <base>COM1_BASE</base>
+        <irq>COM1_IRQ</irq>
+    </legacy_vuart>
+    <legacy_vuart id="1">
+        <type>VUART_LEGACY_PIO</type>
+        <base>COM2_BASE</base>
+        <irq>COM2_IRQ</irq>
+        <target_vm_id>0</target_vm_id>
+        <target_uart_id>1</target_uart_id>
+    </legacy_vuart>
+    <console_vuart id="0">
+        <base>INVALID_PCI_BASE</base>
+    </console_vuart>
+    <communication_vuart id="1">
+        <base>INVALID_PCI_BASE</base>
+        <target_vm_id>1</target_vm_id>
+        <target_uart_id>1</target_uart_id>
+    </communication_vuart>
+  </vm>
+  <vm id="3">
+    <vm_type>POST_STD_VM</vm_type>
+    <guest_flags>
+        <guest_flag>0</guest_flag>
+    </guest_flags>
+    <cpu_affinity>
+        <pcpu_id>0</pcpu_id>
```

Comment:
Is there any reason here we're sharing CPUs with VM 1 instead of giving other vCPUs (we have a total of 10 of them, so still some unallocated ones)

Reply:
This is the default config xml for ADL enabling, refering to other enabled platform. Users can generate their specific config based on this default config for different cpu assignment or other resource assignment.

Reply:
I understand customer can (and should) defined their own config based on their requirements. I'm just surprised that we seem to use a sub-optimal config to begin with. Not a big deal but knowing that default values tend to stick longer than they should, I think it would have been better to define a more balanced or optimized scenario and use that throughout our development.

Reply:
Agree, we'd better to re-define scenario/VM config of how to use 10 cores(2-8) in ADL-P platform, rather than just re-use TGL-4 core definition and leave 6 cores idle.  Welcome more comments and ideas.

Reply:
updated cpu assignment: vm1 (core 0/1), vm2 (core 2/3), vm3-vm7 (share core 4-9), sos default all cores. please review.